.env

```diff
@@ -0,0 +1,5 @@
+DB_HOST='localhost'
+DB_PORT= 5432
+DB_USERNAME= 'postgres'
+DB_PASSWORD= 'Lola.2022'
+DB_NAME='nest-db'
```

Comment:
`.env` files should go in the `.gitignore` since you don't want to expose your credentials